src/module.c

```diff
@@ -883,6 +887,44 @@ void ShardingEvent(RedisModuleCtx *ctx, RedisModuleEvent eid, uint64_t subevent,
     RedisModule_Log(ctx, "verbose", "Successfully executed " #f);              \
   }
 
+int redisMajorVersion;
```

Comment:
static or extern, please

Reply:
actually, let's move this into a new file; `cluster_functions.c` or similar.

Reply:
I actually moved it into a file but forgot to add it...